modules/tensor_mechanics/src/bcs/Pressure.C

```diff
@@ -244,8 +262,40 @@ Pressure::precalculateQpJacobian()
   }
 }
 
+template <>
 void
-Pressure::precalculateQpOffDiagJacobian(const MooseVariableFEBase & /*jvar*/)
+PressureTempl<false>::precalculateQpOffDiagJacobian(const MooseVariableFEBase & /*jvar*/)
 {
   precalculateQpJacobian();
 }
+
+template <>
+void
+PressureTempl<true>::precalculateQpJacobian()
+{
+  mooseError("This should never be called");
```

Comment:
De we even need to implement those specializations? We'd get a runtime linker error if anything tried to use them.

Reply:
That's what I thought.  Maybe I did something wrong, but the tests all died until I added those.

Reply:
Even if I comment out the final two lines (`template class ...`), I get a 'symbol not found' error unless I include those implementations.

Reply:
Any thoughts?  I don't see a way around this.

Reply:
Yeah, I wanted to check something today. And then I'll either approve/merge or make a suggestion.